code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -237,6 +237,9 @@
 	var/warding_aura = 0
 	var/recovery_aura = 0
 
+	///Determines how fast xenos heal
+	var/xeno_blood = 100
```

Comment:
This needs much better explanation.

Reply:
Additionally, please make this some 'larger' number and have the rest of the procs adjusted accordingly.

Humans have 560 units of blood. Xenos are 'usually' bigger.

Gonna harvest that xeno blood for its sweet sweet juices.

Reply:
changed